src/python/pants/option/arg_splitter_test.py

```diff
@@ -10,14 +10,8 @@
 
 import pytest
 
-from pants.option.arg_splitter import (
-    AllHelp,
-    ArgSplitter,
-    NoGoalHelp,
-    ThingHelp,
-    UnknownGoalHelp,
-    VersionHelp,
-)
+from pants.core.register import builtin_goals
```

Comment:
Not great. As the help integration has moved out of arg splitter, the tests for the help integration ought to move as well..